projects/igniteui-angular/src/lib/combo/combo.pipes.ts

```diff
@@ -13,18 +14,21 @@ import { DefaultSortingStrategy } from '../data-operations/sorting-strategy';
     name: 'comboFiltering'
 })
 export class IgxComboFilteringPipe implements PipeTransform {
-    public transform(collection: any[], searchValue: any, displayKey: any, shouldFilter: boolean) {
+    public transform(collection: any[], searchValue: any, displayKey: any,
```

Comment:
Currently, when clicking the `showSearchCaseIcon` while there is already a search criteria typed, the icon toggles, yet the pipe transform method is not called and thus the dropdown list result items are not updated to respect the toggled caseSensitive value.  It is not the major use case, yet one would expect the items are updated in this case as well. Adding gif-s for comparison. 
![showSearchCaseIcon-no-update-on-icon-click-igxCombo](https://user-images.githubusercontent.com/4296944/88273345-bcc62980-cce2-11ea-9f83-70827f9c129c.gif)
![showSearchCaseIcon-update-on-icon-click-igxCombo](https://user-images.githubusercontent.com/4296944/88273350-bd5ec000-cce2-11ea-9c55-048304670c66.gif)

I have played a bit and could force the pipe to work calling it's transform method manually and passing the required arguments:

```
<igx-icon fontSet="case-sensitive" name="case-sensitive" [isActive]="filteringOptions.caseSensitive"
    (click)="**toggleAndUpdate(searchValue, displayKey, filterable, filteringOptions**)">
</igx-icon>
```

```
    public toggleAndUpdate( searchValue, displayKey, filterable, filteringOptions) {
        this.filteringOptions.caseSensitive = !this.filteringOptions.caseSensitive;
        // If you mutate the array, no pipe is invoked and the display isn't updated;
        // if you replace the array, the pipe executes and the display is updated.
        this.data = cloneArray(this.data);
        this.filteringPipe.transform(this.data, searchValue, displayKey, filterable, filteringOptions);
    }
```
Notice, a new data collection should be passed, as change detection will not like the old one being mutated. This is required so the list items actually rerenders with the correct items returned from the pipe.